app/helpers/application_helper.rb

```diff
@@ -71,9 +71,18 @@ def multiple_relationship_link(record, table_name)
         end
       else
         out = content_tag(:li) do
-          link_to("#{plural} (#{count})",
-                  polymorphic_path(@record, :display => table_name.to_s.pluralize),
-                  :title => _("Show %{plural_linked_name}") % {:plural_linked_name => plural})
+          if restful_routed?(record)
+            link_to("#{plural} (#{count})",
+                    polymorphic_path(record, :display => table_name.to_s.pluralize),
+                    :title => _("Show %{plural_linked_name}") % {:plural_linked_name => plural})
+          else
+            link_to("#{plural} (#{count})",
+                    {:controller => controller_name,
+                     :action     => 'show',
+                     :id         => record.id,
+                     :display    => table_name.to_s.pluralize},
+                    :title => _("Show %{plural_linked_name}") % {:plural_linked_name => plural})
```

Comment:
> I assume rubocop is unhappy about the not aligned hash entry...

It's not the alignment that it's not happy about... it's complaining about "Missing curly braces around a hash parameter." -- which I've now added.